.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,58 @@
+name: Build VerseReportingProcessor Docker Image
+
+on:
+  push:
+    paths:
+      - VerseReportingProcessor
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  #build and publish image
+  Build-Image:
+    name: Build API image
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-qemu-action@v2      
+      - uses: docker/setup-buildx-action@v2
+
+      - name: Build API image with buildx
+        uses: docker/build-push-action@v3
+        with:
+          context: ./VerseReportingProcessor
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+  Push-Image:
+    name: Push images if this is a branch
+    runs-on: ubuntu-20.04
```

Comment:
Why not current lts 22.04?

Reply:
I think 22.04 wasn't available as a build image when I wrote what I copy/pasted. Changed to ubutnu-latest which should be good for this build.